packit_service/worker/jobs.py

```diff
@@ -575,11 +575,16 @@ def is_project_public_or_enabled_private(self) -> bool:
         in our service configuration.
 
         Returns:
-            `True`, if the project is public or enabled in our service config,
+            `True`, if the project is public or enabled in our service config
+            or the check is skipped,
             `False` otherwise.
         """
+        # do the check only for events triggering the pipeline
+        if isinstance(self.event, AbstractResultEvent):
```

Comment:
> but isinstance() returns True also for subclasses.

that's the exact reason why I used it, because I want the check to be skipped also for e.g. AbstractCoprBuildEvent